src/Compute/Compute.Autorest/examples/Set-AzVMRunCommand.md

```diff
@@ -1,10 +1,10 @@
 ### Example 1: Simple Example
 ```powershell
-PS C:\> Set-AzVMRunCommand -ResourceGroupName $rgname -VMName $vmname -RunCommandName 'firstruncommand' 
+Set-AzVMRunCommand -ResourceGroupName $rgname -VMName $vmname -Location $location -RunCommandName 'firstruncommand' 
```

Comment:
lets make it -Location "eastus" to match the output below

Reply:
and PS C:\>

Reply:
Fixed, good catch!